frontend/packages/policy-editor/src/components/PolicySummary/FeedbackForm/FeedbackForm.tsx

```
@@ -0,0 +1,54 @@
+import type { ReactElement } from 'react';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { FeedbackFormImpl } from '@studio/feedback-form';
```

ESLint

[error] '@studio/feedback-form' imported multiple times.

(import/no-duplicates)